### PR TITLE
Update enum tests

### PR DESCRIPTION
**Pull request by jkimbo**
Update enum tests to bring them inline with graphql-js v0.13.1

I've not included the complex value tests since I'm not sure how it can be achieved in Python.

### CODE REVIEW

File: graphql/type/definition.py [hunk 1]
```diff
@@ -406,6 +406,12 @@ def __init__(self, name, values, description=None):
 
         self.values = define_enum_values(self, values)
 
+    def getValues(self):
```
Comment (syrusakbary) [inline graphql/type/definition.py]: We should name this `get_values` (camel case)

File: graphql/type/definition.py [hunk 2]
```diff
@@ -406,6 +406,12 @@ def __init__(self, name, values, description=None):
...
+        return self.values
+
+    def getValue(self, name):
```
Comment (syrusakbary) [inline graphql/type/definition.py]: This should be `get_value`

**Comment by syrusakbary** [comment]
I will happily merge the PR once the functions are renamed to `camel_case` 👍

**Comment by jkimbo** [comment]
@syrusakbary done!

**Comment by jkimbo** [comment]
@syrusakbary this should be ready to merge now

**Pull request merged**